policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/basic_personal_amount/applicable_amount.yaml

```diff
@@ -0,0 +1,9 @@
+description: Yukon credits this basic personal amount, based on this applicable amount
+values:
+  2022-01-01: 1_479
+metadata:
+  unit: currency-CAD
+  label: Yukon income applicable amount
+  reference:
+  - title: Worksheet for the Yukon 2023 Personal Tax Credits Return
```

Comment:
add 2022 reference and double check values

Reply:
leave the law reference and add a comment: # The credit amount is not mentioned in the Yukon income tax act

https://laws.yukon.ca/cms/images/LEGISLATION/PRINCIPAL/2002/2002-0118/2002-0118.pdf

Reply:
for every yaml file in this PR